src/wallet/wallet.cpp

```diff
@@ -2815,7 +2815,7 @@ bool CWallet::CreateTransaction(interfaces::Chain::Lock& locked_chain, const std
         LOCK(cs_wallet);
         {
             std::vector<COutput> vAvailableCoins;
-            AvailableCoins(*locked_chain, vAvailableCoins, true, &coin_control);
+            AvailableCoins(*locked_chain, vAvailableCoins, true, &coin_control, 1, MAX_MONEY, MAX_MONEY, 0, coin_control.m_min_depth);
```

Comment:
`AvailableCoins()` could be refactored to not take an `nMinDepth` argument now that coin control has a `m_min_depth` member. Could be done in a follow-up PR.

Reply:
as a larger discussion(for another time), what should be in CCoinControl and what shouldn't be? It's nice to have a compact struct that describes the intent of each payment, including all known constraints by the caller.